esp/src/eclwatch/ESPWorkunit.js

```diff
@@ -312,7 +312,13 @@ define([
             WsWorkunits.WUUpdate({
                 request: request,
                 load: function (response) {
-                    context.updateData(response.WUUpdateResponse.Workunit);
+                    if (response.Exceptions) {
+                        arrayUtil.forEach(response.Exceptions.Exception, function (item, idx) {
+                            context.updateData(response.Exceptions.Exception[idx].Message);
```

Comment:
No, I am passing in the response from WUUpdate which happens to be located at: response.Exceptions.Exception[idx].Message which states ""GMT: Invalid cluster name: ClusterName"